backend/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest==5.4.1
+requests==2.23.0
```

Comment:
A setup.cfg with different variants/features might be better than having multiple requirements.txt. But we can refactor that later